### DISCUSSION TITLE
firefox

### DISCUSSION THREAD

**Discussion by Antwa-sensei253** [The-CodingSloth/haha-funny-leetcode-extension#21] [Q&A]
Hi, How to add it to firefox?
I tried to add install add-on from a file using npm method
but did not work

**Comment by The-CodingSloth** [comment]
Currently this extension only works on chrome. Planning to create the Firefox version soon 👍

**Reply by Antwa-sensei253** [reply]
this page might help: https://extensionworkshop.com/documentation/develop/porting-a-google-chrome-extension/
I will work on this as a side project
